splice_machine/src/main/java/com/splicemachine/derby/impl/store/access/base/SpliceConglomerate.java

```diff
@@ -215,4 +219,29 @@ public boolean equals(Object o){
     public int hashCode(){
         return (int)(containerId^(containerId>>>32));
     }
+
+    @Override
+    public void write(UnsafeRowWriter unsafeRowWriter, int ordinal) throws StandardException {
+        throw new RuntimeException("Not Implemented");
```

Comment:
Don't use RuntimeException to denote not being implemented: use UnsupportedOperationException